README.md

```diff
@@ -81,6 +111,15 @@ $ sudo -E PATH=$PATH make USE_DOCKER=true rootfs
 $ sudo -E PATH=$PATH make USE_DOCKER=true AGENT_INIT=yes rootfs
 ```
 
+#### dracut based rootfs
+
+> Note: the dracut build method does not need a rootfs as a base for an image or initrd.
```

Comment:
Please make the note bold as specified in https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes and capitalise the first word:

> **Note:** The dracut ...

Same comment for notes below.